pom.xml

```diff
@@ -185,7 +185,7 @@
     <paranamer.version>2.8</paranamer.version>
     <maven-antrun.version>1.8</maven-antrun.version>
     <commons-crypto.version>1.0.0</commons-crypto.version>
-    <arrow.version>0.4.0</arrow.version>
+    <arrow.version>0.8.0-SNAPSHOT</arrow.version>
```

Comment:
Please don't forget that we also need to update `dev/deps/spark-deps-hadoop-2.x` files.

Reply:
done